src/dymaptic.GeoBlazor.Core/Model/GeometryEngine.cs

```diff
@@ -0,0 +1,896 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Geometries;
+using dymaptic.GeoBlazor.Core.Objects;
+using Microsoft.Extensions.Configuration;
+using Microsoft.JSInterop;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+
+namespace dymaptic.GeoBlazor.Core.Model;
+
+/// <summary>
```

Comment:
These look great! Do any of them deviate from the JS API?

Reply:
Not really, no. They're very much 1-1 with the exception of having to write out parameter overrides.